src/test/java/com/jakewharton/retrofit2/adapter/rxjava2/FlowableThrowingTest.java

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright (C) 2015 Square, Inc.
```

Comment:
The completable test was written from scratch. These ones were copied from existing source files and adapted which is why they retain their original copyright date.